packages/@aws-cdk/aws-ecs/test/fargate/integ.firelens-cloudwatch.ts

```diff
@@ -0,0 +1,89 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as ecs from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-ecs-integ');
+
+const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 2 });
+
+// grant cloudwatch logs permissions to task role
+const taskIAMRole = new iam.Role(stack, 'firelens-cloudwatch-role', {
```

Comment:
I [added the code](https://github.com/aws/aws-cdk/pull/5457/files#diff-2d9d3b0b1534ae5c6ddea41a79e3c74bR201) to check "awsfirelens" log driver in task definition, it will create default firelens log router container and IAM policies for this task if user didn't add any firelens log router container in the task definition.